experimental/packages/exporter-logs-otlp-proto/test/browser/CollectorLogsExporter.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
We can probably rename this file to `OTLPLogsExporter.test.ts` - the exporters have not been called `CollectorExporter` for a very long time :slightly_smiling_face: